assignment.js

```diff
@@ -32,6 +32,13 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+let multiply = num => {
+    let baseNumber = 0
+    for (let i = 1; i < 12; i++){
```

Comment:
@SOyeniyi801  nice job using a for loop here, code was well structured and easy to follow. 

An example of an alternative approach to this problem using a while loop [can be found here](https://github.com/jsd20200127/hw-01-js-basics-control-flow-loops-solution/blob/master/assignment.js#L38)